src/strategies/uniswap/index.ts

```diff
@@ -0,0 +1,41 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { uni_abi } from './UniV2.json';
```

Comment:
Var names should be camelCased
```suggestion
import { uniAbi } from './UniV2.json';
```